tendrl/monitoring_integration/sync/dashbaord_sync.py

```diff
@@ -18,20 +19,30 @@ def refresh_dashboard(self):
         try:
             # check alert organization is exist
             if NS.config.data["org_id"]:
-                cluster_details = {}
-                dashboards = []
-                integration_ids = utils.get_resource_keys("", "clusters")
-                for integration_id in integration_ids:
-                    key = "/clusters/%s/TendrlContext/sds_name" % \
-                        integration_id
-                    sds_name = etcd_utils.read(key).value
-                    if sds_name == constants.GLUSTER:
-                        cluster_details, dashboards = gluster_cluster_details.\
-                            get_cluster_details(
-                                integration_id
+                all_cluster_details = {}
+                integration_ids = self.get_managed_clusters_integration_id()
+                if integration_ids:
+                    for integration_id in integration_ids:
+                        key = "/clusters/%s/TendrlContext/sds_name" % \
+                            integration_id
+                        sds_name = etcd_utils.read(key).value
+                        if sds_name == constants.GLUSTER:
+                            gluster_cluster_details.get_cluster_details(
+                                integration_id,
+                                all_cluster_details
```

Comment:
What is this variable passed for? The function is called `get_cluster_details` and it doesnt return anything?

Reply:
i will modify this